CMakeLists.txt

```diff
@@ -194,3 +194,12 @@ if (LOG_INFO)
         PRIVATE HAVE_LOG_NO_INFO=0
         )
 endif()
+
+###############################################################################
+# Install Resources
+###############################################################################
+# Install example files
+
+INSTALL(
+    DIRECTORY ${PROJECT_SOURCE_DIR}/resources/configurations/examples/
+    DESTINATION ${BIN_INSTALL_DIR}/resources/)
```

Comment:
Perhaps this location is not the best one, it appears a bit hidden to me.

Reply:
:+1: